spec/models/job_proxy_dispatcher_get_eligible_proxies_for_job_spec.rb

```diff
@@ -21,6 +21,9 @@
 
       context "with a vm scan job, " do
         before(:each) do
+          # admin user is needed to process Events
+          FactoryGirl.create(:user_with_group, :userid => "admin")
+          MiqEventDefinition.stub(:find_by_name => true)
```

Comment:
I'm finding the copy-paste of this pattern very interesting...what do all these changes do? cc @matthewd